techniques/css/C39.html

```diff
@@ -2,102 +2,76 @@
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
-         </p>
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://w3c.github.io/wcag/understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。</p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>Content using technologies that support <a href="https://www.w3.org/TR/CSS/">Cascading Style Sheets (CSS)</a>.
-            </p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/animation-from-interactions">Success Criterion 2.3.3: Animation from Interactions</a> (Sufficient)</span>.
-            </p>
+            <h2>適用 (対象) </h2>
+            <p><a href="https://www.w3.org/TR/CSS/">Cascading Style Sheets (CSS)</a> をサポートするウェブコンテンツ技術を使用したコンテンツ</p>
+            <p>この達成方法は、<span><a href="https://waic.jp/docs/WCAG21/Understanding/animation-from-interactions.html">達成基準 2.3.3: インタラクションによるアニメーションを理解する</a> (十分) </span>に関連する。</p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to allow users to prevent animation from being
-               displayed on Web pages, via the use of of the 'prefers-reduced-motion' CSS Media Query.
-            </p>
-            <p>Some users experience distraction or nausea from animated content. For example, if
-               scrolling a page causes elements to move (other than the essential movement associated
-               with scrolling) it can trigger vestibular disorders. Enclosing the CSS that creates
-               the animations in a media query allows people to prevent those symptoms.
-            </p>
-            <p>A typical example is 'parallax scrolling', where backgrounds move at different rates.
-               The movement due to scrolling the page is essential (and under the users control),
-               but additional movement triggered by the scrolling can also trigger vestibular symptoms.
-            </p>
-            <p>The understanding document for <a href="../../understanding/motion-actuation.html#resources">Motion Actuation</a> includes links for changing the reduce motion setting.
-            </p>
+            <h2>解説</h2>
+            <p>この達成方法は 'prefers-reduced-motion' CSS メディアクエリを使用して、ユーザーが Web ページのアニメーションを非表示にできることを目的とする。</p>
```

Comment:
MDNを読む限り、
https://developer.mozilla.org/ja/docs/Web/CSS/@media/prefers-reduced-motion

あくまで減らすのであって、なくすわけではないので、非表示は誤りではと。（タイトルでは防止としていますし）
https://ejje.weblio.jp/content/prevent を見ながら以下のようにしてみました。

user：利用者
web：ウェブ

```suggestion
            <p>この達成方法は 'prefers-reduced-motion' CSS メディアクエリを使用して、利用者にウェブページでアニメーションが表示されないようにすることを目的とする。</p>
```